packages/cozy-harvest-lib/src/connections/files.js

```diff
@@ -0,0 +1,39 @@
+const FILES_DOCTYPE = 'io.cozy.files'
+
+/**
+ * Creates a directory from a given path
+ * @param  {Object}  client CozyClient
+ * @param  {string}  path   Directory path
+ * @return {Object}         Directory attributes
+ */
+export const createDirectoryByPath = async (client, path) => {
+  const { data } = await client
+    .collection(FILES_DOCTYPE)
+    .createDirectoryByPath(path)
+  return data
+}
+
+/**
+ * Retrieves a directory from its path
+ * @param  {Object}  client CozyClient
+ * @param  {string}  path   Directory path
+ * @return {Object}        Created io.cozy.files document
+ */
+export const fetchDirectoryByPath = async (client, path) => {
```

Comment:
Why not statDirectoryByPath ? Since you are calling statByPath below ?

Reply:
I think the `statBy` is pretty unexplicit, so I tend it to encapsulate it into functions with a better name. But you're right, let's be logic and keep this one.

Reply:
If your function wraps the statByPath to add a try/catch for 404, I think it is confusing to have a different name, a reader of the function has to forget cannot rely on its the knowledge of the underlying library.

To me, either we find the name of library not explicit and we send a PR there to deprecate the confusing name and use the new name, or we accept the name of the library and keep the coherence :)